### DISCUSSION TITLE
rec导出模型报编码错误

### DISCUSSION THREAD

**Discussion by alanxinn** [PaddlePaddle/PaddleOCR#14209] [Q&A]
paddlepaddle==2.6.1
paddleocr==2.9.1
 E:\Desktop\PaddleOCR>python tools/export_model.py -c output\rec_ppocr_v4\config.yml -o Global.pretrained_model=output\rec_ppocr_v4\latest  Global.save_inference_dir=output\rec_ppocr_v4\best_model_infer

UnicodeEncodeError: 'gbk' codec can't encode character '\xcb' in position 0: illegal multibyte sequence

**Comment by UserWangZz** [comment]
这个是系统的编码问题吧？
